fuse_core/test/test_timestamp_manager.cpp

```diff
@@ -112,9 +112,12 @@ TEST_F(TimestampManagerTestFixture, Empty)
   EXPECT_EQ(ros::Time(20, 0), *stamp_range_iter);
 
   // Verify the expected queries were performed
-  ASSERT_EQ(1ul, generated_time_spans.size());
```

Comment:
I have to update the tests here because now `TimestampManager::query` calls the `generator_` for the `last_stamp`, which adds a `(last_stamp, last_stamp)` time span add the end.

Note that in the actual generator in `Unicycle2D` the state history is a map with the time stamp as a key, so only a single entry is added per stamp.